<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSeriesDataWidgetForm</class>
 <widget class="QWidget" name="TimeSeriesDataWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="m_valueLabel">
       <property name="text">
        <string>Value:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_timeComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_timeLabel">
       <property name="text">
        <string>Date &amp; Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_valueComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_idComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_idLabel">
       <property name="text">
        <string>Id:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
